lib/screens/explore_screens/unplant_seeds/unplant_seeds_screen.dart

```diff
@@ -94,11 +94,14 @@ class UnplantSeedsScreen extends StatelessWidget {
                             AlertInputValue('Need to keep at least 5 planted seeds',
                                 isVisible: state.showMinPlantedBalanceAlert),
                             const SizedBox(height: 60),
-                            ClaimUnplantSeedsBalanceRow(
-                                onTapClaim: () {},
-                                isClaimButtonEnable: false,
-                                tokenAmount: TokenDataModel(0),
-                                fiatAmount: FiatDataModel(0)),
+                            if (state.showUnclaimedBalance)
+                              ClaimUnplantSeedsBalanceRow(
+                                  onTapClaim: () {},
+                                  isClaimButtonEnable: false,
+                                  tokenAmount: TokenDataModel(0),
+                                  fiatAmount: FiatDataModel(0))
+                            else
+                              const SizedBox.shrink(),
```

Comment:
Not sure if the `else` clause its nessesary here It make sense when you have another different widget to show.

```
children:[
if(someVal)
myWidget()
]

```
In the previus sniped if `someVa`l is false nothing is drawed (same effect of `SizedBox.shrink()`).

Reply:
I dont think we had 'if' 'else' before we had to use ? : , this got me a little confuse but your logic is right I only need the 'if' . Fixed